packages/apps-config/src/endpoints/productionRelayKusama.ts

```diff
@@ -313,6 +313,16 @@ export const prodParasKusama: EndpointOption[] = [
       Parallel: 'wss://heiko-rpc.parallel.fi'
     }
   },
+  {
+    info: 'heiko',
```

Comment:
```suggestion
    info: 'heiko2',
```

I'm guessing this is what CI complains about (have not checked)